public/js/pimcore/document/document.js

```diff
@@ -87,6 +87,11 @@ pimcore.document.document = Class.create(pimcore.element.abstract, {
             return;
         }
 
+        if (this.saveInProgress()){
+            pimcore.helpers.showNotification(t("warning"), t("Another saving process is in progress, please wait and retry again"), "info", '');
```

Comment:
```suggestion
            pimcore.helpers.showNotification(t("warning"), t("Another saving process is in progress, please wait and retry again"), "info");
```
4th argument only used for error type